scannerlib/configchecks/group_criteria.go

```diff
@@ -215,3 +223,65 @@ func getCmpStr(gc *ipb.GroupCriterion) string {
 	}
 	return strconv.Itoa(int(gc.GetConst()))
 }
+type lessThanVersionMatcher struct {
+	cmpStr string
+}
+func (m *lessThanVersionMatcher) match(group string) bool {
+	if !swVersRe.MatchString(group) {
+		log.Printf("unable to parse %q as a software version", group)
+		return false
+	}
+	version := strings.ReplaceAll(m.cmpStr, "-", ".")
+	re := regexp.MustCompile("\\W+")
+	chunks_group := re.Split(group, -1)
```

Comment:
use camelCase and a name that clarifies this is the version found in the file - e.g. chunksDetectedVersion